helm/primehub/values.yaml

```diff
@@ -44,6 +44,9 @@ jupyterhub:
       - containerPort: 9102
       - containerPort: 9125
         protocol: UDP
+    extraConfig:
+      primehub: |
+        ## jupyterhub_primehub.py
```

Comment:
Can we make the comment more verbose? such as saying the .py file is inlined while chart building, and for dev installing from directory, use --set-file

Reply:
Resolved